src/pages/Quiz/index.js

```diff
@@ -54,54 +37,54 @@ const useStyles = makeStyles((theme) => ({
 
 const Quiz = (props) => {
   const classes = useStyles();
-  const {
-    quiz = [],
-    common: { init = false, error = null },
-  } = props;
+  const { quiz = [] } = props;
 
-  if (init) {
-    props.fetchQuestions();
-  }
+  useEffect(() => {
+    props.fetchFirebaseData();
```

Comment:
Why do you need to fetch firebase data here?

Reply:
You're right, I don't need to use fetchFirebaseData here. Updated in last commit